### ISSUE TITLE
Editing Keywords

### ISSUE THREAD

**Issue by NANNERGTO** [dobbelina/repository.dobbelina#983]
Hi there everyone. 
Is there an option to edit existing keywords  in Cumination you have searched for in the past? I would like to edit a misspelled keyword or add to it , instead of adding a whole new keyword to the list. 
i.e. _**Annamolli**_ but edit it to _**Anna-molli**_ 

Thanks in advance.

**Comment by dobbelina** [comment]
Will be in the next update https://github.com/dobbelina/repository.dobbelina/commit/8118d2190c56fdaabded846a0ba62fa8117725ca

**Comment by DJMix-I-Can** [comment]
What's up guys/girls.... Whomever provided this fix, can you double check that the thumbnails are still showing up after adding it via context menu, "Add Item To...", then "Main Menu". All of the logos from the sites that I've added to my main menu have disappeared. They're blank but still show their name. I'm still able to go into that site but the sites logo doesn't show. I thought it was an error on my end by editing the py files myself but double checked, redid it & it's still happening. It happens on my Android Nvidia Shield & Windows 7 laptop. Thanks...

**Comment by 12asdfg12** [comment]
I don't think this has anything to do with the fix.
Is the image missing even if you re-add a site (or add a new site) to the main menu?
Can you check the content of the column "image" from "favorites.db", table "custom_listitem"? Is the image path correct?

**Comment by DJMix-I-Can** [comment]
> I don't think this has anything to do with the fix.
> Is the image missing even if you re-add a site (or add a new site) to the main menu?
> Can you check the content of the column "image" from "favorites.db", table "custom_listitem"? Is the image path correct?

What's up @12asdfg12 .  I did add a new site that wasn't originally on my main menu just to make sure. The icon didn't load. I'll double-check tomorrow & check the favorites.db table. Thanks for your feedback.

**Comment by dobbelina** [comment]
A fix will be available in the next release https://github.com/dobbelina/repository.dobbelina/commit/7a8790cb7c4d831c3ec6cb4907b5a719609e3f52

**Comment by DJMix-I-Can** [comment]
@12asdfg12 I checked the favorites.db & the image path was there for all of them. I even tried a back up of my favorites.db file & still wouldn't show. I just wound up removing them & re-adding them again. I did this on my Nvidia Shield. As for the laptop, I added the above fix & some of them reappeared but not all of them. I had to remove & re-add them as well. Not sure what happened but thanks for your help.

**Comment by 12asdfg12** [comment]
The fix corrects the path for images in case "favorites.db" is moved between different Kodi installations.
If an image is still missing, you must check if the image form "favorites.db" exists in the image folder (or if the image URL works).

**Comment by DJMix-I-Can** [comment]
I definitely understand what your stating. I grabbed the old favorites.db file that wasn't working & compared it to the new one after removing & re-adding the sites. They were exactly the same. What I did notice is that the icons that did show were the ones which had a URL image link for it. The icons that didn't show was for those that had the images stored withing the images folder within cumination.
